### PR TITLE
Section fix

### LINKED ISSUE #19: Standard Sections
- fonts
- title/heading styles
- text styles
- link styles (global)
- margins/padding

How it looks on https://www.choicehotels.com/explore/road-trips/best-winter-road-trips
Screenshot 2023-02-09 at 08 29 00
...
Screenshot 2023-02-09 at 08 28 32

### PR DESCRIPTION
**Pull request by cuzuco2**
Please review

Fix #19

Test URLs:
- Before: https://main--choice--hlxsites.hlx.page/en-us/explore/best-winter-road-trips
- After: https://section-fix--choice--hlxsites.hlx.page/en-us/explore/best-winter-road-trips

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -238,6 +238,16 @@ main .section {
   padding-right: var(--spacing-s);
 }
 
+main .section h2{
+  font-size: var(--heading-font-size-l);
+  font-weight: 400;
```
Comment (dzehnder) [inline styles/styles.css:243]: please use var

File: styles/styles.css [hunk 2]
```diff
@@ -315,7 +316,7 @@ main .default-content-wrapper {
   }
 
   main .default-content-wrapper {
-    width: 75%;
+    width: 75%;
```
Comment (dzehnder) [inline styles/styles.css:319]: remove spaces after `;`

File: styles/styles.css [hunk 3]
```diff
@@ -238,6 +238,16 @@ main .section {
...
 }
 
+main .section h2{
+  font-size: var(--heading-font-size-l);
+  font-weight: var(--font-weight-normal);
+  letter-spacing: -0.1px;
+}
+
+main .section a {
+text-decoration: underline;
```
Comment (rofe) [inline styles/styles.css:248]:
```suggestion
  text-decoration: underline;
```

File: styles/styles.css [hunk 4]
```diff
@@ -238,6 +238,16 @@ main .section {
   padding-right: var(--spacing-s);
 }
 
+main .section h2{
+  font-size: var(--heading-font-size-l);
+  font-weight: var(--font-weight-normal);
+  letter-spacing: -0.1px;
```
Comment (rofe) [inline styles/styles.css:244]: Hmm... looks hacky and would call for a code comment 😉

**Review by rofe — DISMISSED** [review]

File: styles/styles.css [hunk 5]
```diff
@@ -238,6 +238,16 @@ main .section {
   padding-right: var(--spacing-s);
 }
 
+main .section h2{
```
Comment (solaris007) [inline styles/styles.css:241]:
```suggestion
main .section h2 {
```

**Review by solaris007 — DISMISSED** [review]

**Review by solaris007 — APPROVED** [review]

**Pull request merged**